### ISSUE TITLE
AppImage runs spew QFactoryLoader log messages

### ISSUE THREAD

**Issue by ferdnyc** [OpenShot/openshot-qt#1777]
**Describe the bug**
Runs of OpenShot from the AppImage are displaying incredibly verbose `QFactoryLoader` messages on the console. I _believe_ this is because `usr/bin/openshot-qt` in the AppImage is setting `QT_DEBUG_PLUGINS=1`

**System Details:**
 - Operating System / Distro: Fedora 28
 - OpenShot Version 2.4.2 from `OpenShot-v2.4.2-x86_64.AppImage`

**To Reproduce**
1. Execute `OpenShot-v2.4.2-x86_64.AppImage` in a terminal window
2. Respond to the "Create a shortcut?" question
3. Read console output

**Expected behavior**
Only pertinent log messages are output

**Actual behavior**
```console
$ ./OpenShot-v2.4.2-x86_64.AppImage
Loaded modules from current directory: /tmp/.mount_tPV6nB/usr/bin
      launch:INFO ------------------------------------------------
      launch:INFO    OpenShot (version 2.4.2)
      launch:INFO ------------------------------------------------
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/bin/platforms" ... 
QFactoryLoader::QFactoryLoader() looking at "/tmp/.mount_tPV6nB/usr/bin/platforms/libqeglfs.so" 
Found metadata in lib /tmp/.mount_tPV6nB/usr/bin/platforms/libqeglfs.so, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.2",
    "MetaData": {
        "Keys": [
            "eglfs"
        ]
    },
    "className": "QEglFSIntegrationPlugin",
    "debug": false,
    "version": 328193
}


Got keys from plugin meta data ("eglfs") 
QFactoryLoader::QFactoryLoader() looking at "/tmp/.mount_tPV6nB/usr/bin/platforms/libqkms.so" 
Found metadata in lib /tmp/.mount_tPV6nB/usr/bin/platforms/libqkms.so, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.2",
    "MetaData": {
        "Keys": [
            "kms"
        ]
    },
    "className": "QKmsIntegrationPlugin",
    "debug": false,
    "version": 328193
}


Got keys from plugin meta data ("kms") 
QFactoryLoader::QFactoryLoader() looking at "/tmp/.mount_tPV6nB/usr/bin/platforms/libqlinuxfb.so" 
Found metadata in lib /tmp/.mount_tPV6nB/usr/bin/platforms/libqlinuxfb.so, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.2",
    "MetaData": {
        "Keys": [
            "linuxfb"
        ]
    },
    "className": "QLinuxFbIntegrationPlugin",
    "debug": false,
    "version": 328193
}


Got keys from plugin meta data ("linuxfb") 
QFactoryLoader::QFactoryLoader() looking at "/tmp/.mount_tPV6nB/usr/bin/platforms/libqminimal.so" 
Found metadata in lib /tmp/.mount_tPV6nB/usr/bin/platforms/libqminimal.so, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.2",
    "MetaData": {
        "Keys": [
            "minimal"
        ]
    },
    "className": "QMinimalIntegrationPlugin",
    "debug": false,
    "version": 328193
}


Got keys from plugin meta data ("minimal") 
QFactoryLoader::QFactoryLoader() looking at "/tmp/.mount_tPV6nB/usr/bin/platforms/libqminimalegl.so" 
Found metadata in lib /tmp/.mount_tPV6nB/usr/bin/platforms/libqminimalegl.so, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.2",
    "MetaData": {
        "Keys": [
            "minimalegl"
        ]
    },
    "className": "QMinimalEglIntegrationPlugin",
    "debug": false,
    "version": 328193
}


Got keys from plugin meta data ("minimalegl") 
QFactoryLoader::QFactoryLoader() looking at "/tmp/.mount_tPV6nB/usr/bin/platforms/libqoffscreen.so" 
Found metadata in lib /tmp/.mount_tPV6nB/usr/bin/platforms/libqoffscreen.so, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.2",
    "MetaData": {
        "Keys": [
            "offscreen"
        ]
    },
    "className": "QOffscreenIntegrationPlugin",
    "debug": false,
    "version": 328193
}


Got keys from plugin meta data ("offscreen") 
QFactoryLoader::QFactoryLoader() looking at "/tmp/.mount_tPV6nB/usr/bin/platforms/libqxcb.so" 
Found metadata in lib /tmp/.mount_tPV6nB/usr/bin/platforms/libqxcb.so, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.2",
    "MetaData": {
        "Keys": [
            "xcb"
        ]
    },
    "className": "QXcbIntegrationPlugin",
    "debug": false,
    "version": 328193
}


Got keys from plugin meta data ("xcb") 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/platforms" ... 
loaded library "/tmp/.mount_tPV6nB/usr/bin/platforms/libqxcb.so" 
loaded library "Xcursor" 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/bin/platformthemes" ... 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/platformthemes" ... 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/bin/platforminputcontexts" ... 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/platforminputcontexts" ... 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/bin/generic" ... 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/generic" ... 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/bin/styles" ... 
QFactoryLoader::QFactoryLoader() checking directory path "/tmp/.mount_tPV6nB/usr/styles" ... 
         app:INFO openshot-qt version: 2.4.2
```

**Comment by ferdnyc** [comment]
> I believe this is because usr/bin/openshot-qt in the AppImage is setting QT_DEBUG_PLUGINS=1

A conjecture reinforced by the fact that running from the source tree with that variable set results in the same spew.
```console
$ QT_DEBUG_PLUGINS=1 src/launch.py
(Same crazy-verbose output)
```

**Comment by ferdnyc** [comment]
Closed by #1778
